src/table/table.component.tsx

```diff
@@ -154,6 +163,22 @@ const Table = React.memo((props: TableProps): React.ReactElement => {
                           ref={provided.innerRef}
                           sx={{ display: 'flex', flexDirection: 'row' }}
                         >
+                          <MuiTableCell
+                            padding="checkbox"
+                            sx={{ width: '48px' }}
```

Comment:
Removing these manually set widths doesn't affect the styling at all as far as I can see, so we should probably remove them